src/components/bio.js

```diff
@@ -4,13 +4,15 @@ import React from "react"
 import styles from "./bio.module.scss"
 
 const Bio = ({ date, author, pinned }) => {
-  const githubUrl = author.github
-    ? `https://github.com/${author.github}.png?size=50`
+  const authorImg = author.featuredImage
+    
+  const imageName = authorImg
+    ? require(`../../content/assets/images/${authorImg}`) 
```

Comment:
Instead of fetching it using require, you should take the help of `gatsby-image`, Checkout [this file](https://github.com/LoginRadius/engineering-portal/blob/master/src/components/card.js) to understand how to use it.

Reply:
Hey @mohammed786 I am following this  [**Rendering images sourced from YAML**](https://www.gatsbyjs.com/tutorial/gatsby-image-tutorial/) but somehow unable to get  frontmater folder for authorImg key like that we have cover img. Can u suggest something that i am missing

Reply:
This is done